src/styles/components/_menu.scss

```diff
@@ -1,10 +1,31 @@
-$interface-types: (
```

Comment:
@rebeccamadsen There are two helper functions in UI, `color()/color-set()` and `palette()/palette-set()`, color defines named colors, and palette attaches those colors to named properties. e.g.

```scss
// in UI
@include color-set('green', #00FF00);
@include palette-set('node', color('green'));
```
```scss
// in Network Canvas
.node {
  color: palette('node');  // #00FF00
}
```

Ideally in Network Canvas we should use the `palette()` method, and define them with `@include palette-set()` in UI.

You can see a list of the presets here, please any that you need:
https://github.com/codaco/Network-Canvas-UI/blob/master/src/styles/global/_default.scss#L29,L47